views/trust-legal-entity-beneficial-owner.html

```diff
@@ -117,7 +117,7 @@ <h1 class="govuk-heading-xl govuk-!-margin-0">{{ pageParams.title }}</h1>
 
         {% set fieldParam = {
           name: 'governingLaw',
-          id: 'law_governed',
```

Comment:
I think these renamings will cause issues with the legal entity change links on the check-answers-page. They will stop the change links linking the correct fields. May need to update them if you need to do these renamings. 

Also, if you are renaming the `id` to the same value as the `name` then you don't need to specify the `id`. If the `id` field is not set then it is set to the value of the `name` field. See inside `includes/inputs/text-input.html`

Reply:
I also fixed the check-answers-page as well. I think it should be fine now